pkg/proxy/metrics/metrics.go

```diff
@@ -265,6 +266,15 @@ var (
 		},
 		[]string{"traffic_policy"},
 	)
+
+	// localhostNodePortsAcceptedPacketsDescription describe the metrics for the number of packets accepted
+	// by iptables which were destined for nodeports on loopback interface.
+	localhostNodePortsAcceptedPacketsDescription = metrics.NewDesc(
+		"kubeproxy_iptables_localhost_nodeports_accepted_packets_total",
+		"Number of packets accepted on nodeports of loopback interface",
+		nil, nil, metrics.ALPHA, "")
```

Comment:
Just confirming: ALPHA means you intend to graduate this metric to BETA and then to GA in the next few releases.  If this is one you are using to develop or debug, and don't intend to promote, use INTERNAL instead.

Reply:
It's not internal; this is for user to use (to figure out if they are depending on the localhost nodeports functionality that iptables kube-proxy implements but nftables kube-proxy does not).

As for graduation; _all_ of the kube-proxy metrics are alpha and there has never been any discussion about graduating any of them :grimacing: